virtual_ship/instruments/ship_underwater_st.py

```diff
@@ -42,26 +49,31 @@ def simulate_ship_underwater_st(
     particleset = ParticleSet.from_list(
         fieldset=fieldset,
         pclass=_ShipSTParticle,
-        lon=0.0,  # initial lat/lon are irrelevant and will be overruled later.
+        lon=0.0,  # initial lat/lon are irrelevant and will be overruled later
         lat=0.0,
         depth=depth,
         time=0,  # same for time
     )
 
     # define output file for the simulation
-    out_file = particleset.ParticleFile(
-        name=out_file_name,
-    )
+    # the default outputdt is good(infinite), as we want to just want to write at the end of every call to 'execute'
+    out_file = particleset.ParticleFile(name=out_file_name)
 
+    # iterate over each points, manually set lat lon time, then
+    # execute the particle set for one step, performing one set of measurement
     for point in sample_points:
         particleset.lon_nextloop[:] = point.location.lon
         particleset.lat_nextloop[:] = point.location.lat
-        particleset.time_nextloop[:] = point.time
+        particleset.time_nextloop[:] = fieldset.time_origin.reltime(
+            np.datetime64(point.time)
```

Comment:
Would it make sense to use `np.datetime64` all throughout VirtualShip? That is what we tend to use in `parcels` too. Or would you prefer to keep `datetime`?

Reply:
to me it is all the same. I tried to remain consistent with datetime, but we can swap to np.datetime64.